geojson_pydantic/features.py

```diff
@@ -15,9 +15,9 @@
 class Feature(GenericModel, Generic[Geom, Props]):
     """Feature Model"""
 
-    type: str = Field(default="Feature", const=True)
-    geometry: Optional[Geom] = None
-    properties: Optional[Props] = None
+    type: Literal["Feature"]
+    geometry: Union[Geom, None] = Field(...)
+    properties: Props = Field(...)
```

Comment:
TO BE DISCUSSED

Reply:
> A Feature object has a member with the name "properties".  The value of the properties member is an object (any JSON object or a JSON null value).
Probably should be: `properties: Union[Props, None] = Field(...)`

Reply:
right so we should accept `properties=None`